### PR TITLE
Fixed #3725 - Email Signature UTF8

### LINKED ISSUE #3725: after upgrade to 7.9.1 email signature in compose view is missing
If an user compose an email by tinymce editor, the present signature is missing after upgrade from 7.9.0 to 7.9.1

More detailed:
played around with the hotfix branch a few days ago, there was a fix for that issue and it works. I implemented an upgraded tinymce editor for the Email-Modul and setup more features for it. This works also. Afte some fixes and after the upgrade, the signature is missing again. 
Just thought I made the mistake by changing  tinymce conf in EmailsComposeView.js, SugarTinyMCE.php, and UserSignatureEditView.html to connect the new tinymce. 

Have a clean installation of 7.9 on a little NAS with very rare own changings. The 7.9.1 patch recorded this morning shows, that the signature is no longer displayed there either. 

Now I am no more sure, if this is my fault or a little bug? 

What should happen?
The signature of every user should available in every compose view in every mode, plain or html.

Actual Situation:
The compose view shows a blank editor area without the individual user signature.

Possible Fix:
Maybe something in the Users Modul?

Steps to Reproduce:
1. Go to Emails -> Compose
2. Compose Email Body window opens. It shows the expected tinymce editor but without the signature
3. Same behaviour in the popup compose view

Environment:

* SuiteCRM 7.9.1 Update
* Firefox (64-bit)):
* mysql 5.6
* centOs 6.8
* php 7.0 /php-fpm

### PR DESCRIPTION
**Pull request by daniel-samson**
## Description

This change fixes the issue with signatures when unsupported UTF-8 characters break the json_encode method, causing the user to not see their signature.

## Motivation and Context

* Fixes the issue

## How To Test This

* Use a PHP 5.3 environment
* Create a signature
* Set the signature up
* Compose an email

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

File: modules/Emails/Compose.php [hunk 1]
```diff
@@ -69,7 +69,7 @@ function initFullCompose($ret)
         echo $composeOut;
     } else {
         //For normal full compose screen
-        include('modules/Emails/index.php');
+//        include('modules/Emails/index.php');
```
Comment (samus-aran) [inline modules/Emails/Compose.php]: Commented out? :scream:

**Comment by samus-aran** [comment]
This is reference to 3725

**Comment by pgorod** [comment]
@daniel-samson I expect this also fixes #3654 ? I noticed you removed the offending line.

I never tested just removing the line because I thought it had to be replaced with some new thing. :-)

**Comment by daniel-samson** [comment]
Hey @pgorod I removed the line because it was creating PHP notice in the error logs. I believe we have still have to look into the email quotation issue. It's on the TODO list. :D

**Pull request merged**